nekoyume/cli.py

```diff
@@ -127,12 +138,17 @@ def repair():
     db.session.commit()
 
 
-cli.add_command(init)
-cli.add_command(neko)
-cli.add_command(shell)
-cli.add_command(sync)
-cli.add_command(doctor)
-cli.add_command(repair)
+@cli.command()
+@click.option('--host',
+              default='127.0.0.1',
+              help='Host to listen')
+@click.option('--port',
+              type=click.IntRange(0, 65535),
+              metavar='PORT',
+              default=5000,
+              help='Port number to listen')
+def dev(host: str, port: int) -> None:
+    app.run(debug=True, port=port)
```

Comment:
this app isn't accepting `host` parameter.

Reply:
I don't get it; could you elaborate on this?

Reply:
I got it.  I didn't use the parameter.